hi-ml-multimodal/src/health_multimodal/image/model/model.py

```diff
@@ -139,7 +158,7 @@ def get_patchwise_projected_embeddings(self, input_img: torch.Tensor, normalize:
         :returns projected_embeddings: tensor of embeddings in shape [batch, n_patches_h, n_patches_w, feature_size].
         """
         assert not self.training, "This function is only implemented for evaluation mode"
-        outputs = self.forward(input_img)
+        outputs = self.forward(ImageModelInput(image=input_img))
```

Comment:
Similar to the comment above, let's split this up into different lines.

Reply:
[Resolving as it's now outdated]